content/zh-cn/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -118,7 +118,7 @@ Before Pods in Kubernetes can be configured to use GMSAs, the desired GMSAs need
 [Windows GMSA 文档](https://docs.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/getting-started-with-group-managed-service-accounts#BKMK_Step1)
 中描述的那样先在活动目录中准备好期望的 GMSA。
 Windows 工作节点（作为 Kubernetes 集群的一部分）需要被配置到活动目录中，以便
-访问与期望的 GSMA 相关联的秘密凭据数据。这一操作的描述位于
+访问与期望的 GSMA 相关联的密钥凭据数据。这一操作的描述位于
```

Comment:
```suggestion
访问与期望的 GSMA 相关联的秘密凭据数据。这一操作的描述位于
```
The original is better. Normally, key is 密钥, and secret is 秘密. If changed to 密钥, it may involve with too many topics to modify.

Reply:
Thanks